plugins/alerts/frontend/public/stylesheets/vue-main.scss

```diff
@@ -0,0 +1,56 @@
+#alerts-view {
+    .text-heading.title {
+        font-size: 13px;
+    }
+
+    .cly-vue-alerts-table {
+        .is-option-button {
+            visibility: hidden;
+            .el-select {
+                -webkit-box-shadow: unset;
```

Comment:
modifying core components should be avoided.
unless this case is an exception.
maybe try to remove this css and see if the design consistency is maintained.